va/va.go

```diff
@@ -330,17 +339,39 @@ func maxAllowedFailures(perspectiveCount int) int {
 	return 2
 }
 
+// mpicSummary is returned by remoteDoDCV and contains a summary of the
+// validation results for logging purposes.
+type mpicSummary struct {
+	// PassedPerspectives are the distinct perspectives that PassedPerspectives validation.
+	PassedPerspectives []string `json:"passedPerspectives"`
+
+	// FailedPerspectives are the disctint perspectives that FailedPerspectives validation.
+	FailedPerspectives []string `json:"failedPerspectives"`
+
+	// PassedRIRs are the distinct Regional Internet Registries that passing
+	// perspectives belonged to.
+	PassedRIRs []string `json:"passedRIRs"`
+
+	// QuorumResult is the Multi-Perspective Issuance Corroboration quorum
+	// result, per BRs Section 5.4.1, Requirement 2.7 (i.e., "3/4" which should
+	// be interpreted as "Three (3) out of four (4) attempted Network
+	// Perspectives corroborated the determinations made by the Primary Network
+	// Perspective).
+	QuorumResult string `json:"quorumResult"`
+}
+
 // Used for audit logging
 type verificationRequestEvent struct {
 	ID                string         `json:",omitempty"`
 	Requester         int64          `json:",omitempty"`
 	Hostname          string         `json:",omitempty"`
 	Challenge         core.Challenge `json:",omitempty"`
 	ValidationLatency float64
-	Error             string `json:",omitempty"`
-	InternalError     string `json:",omitempty"`
-	Perspective       string `json:",omitempty"`
-	RIR               string `json:",omitempty"`
+	Error             string       `json:",omitempty"`
+	InternalError     string       `json:",omitempty"`
+	Perspective       string       `json:",omitempty"`
+	RIR               string       `json:",omitempty"`
+	MPICSummary       *mpicSummary `json:",omitempty"`
```

Comment:
MPIC summary must always be logged, even if it's empty because we caught a local failure. Our logs don't distinguish between a local problem and a remote problem so it's useful to positively state that we did not attempt MPIC.

Reply:
Good point, will change this to not `omitempty`.

Reply:
Should we document why it's not `omitempty`?